cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,168 @@
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the NINETH paragraph with Order List starting with-9. What happens when YOU do not provide necessary Personal Data?", (dataType = "paraPolicyCxWithOrdList", ParaNum = 9) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the TENTH paragraph with Order List starting with-10. With Whom We Share Your Personal Data?", (dataType = "paraPolicyCxWithOrdList", ParaNum = 10) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate the PRIVACY POLICY content for the ELEVENTH paragraph with Order List starting with-11. How long We store your Personal Data?", (dataType = "paraPolicyCxWithOrdList", ParaNum = 11) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the TWELVETH paragraph with Order List starting with-12. Where We Store Your Personal Data", (dataType = "paraPolicyCxWithOrdList", ParaNum = 12) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate the PRIVACY POLICY content for the THIRTEENTH paragraph with Order List starting with-13. How We Protect Your Personal Data", (dataType = "paraPolicyCxWithOrdList", ParaNum = 13) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the FOURTEENTH paragraph with Order List starting with-14. Contact Us", (dataType = "paraPolicyCxWithOrdList", ParaNum = 14) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the FIFTEENTH paragraph with Order List starting with-15. Changes to Our Privacy Policy", (dataType = "paraPolicyCxWithOrdList", ParaNum = 15) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate Date of Document created", () => {
+      privacyPolicy
+        .getDateDocumented()
+        .invoke("text")
+        .should("eq", "April 23rd, 2020");
```

Comment:
Maybe the date should be defined as a constant at the top of this spec instead of being hardcoded here.

Reply:
Done:)